client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/package-row.js

```diff
@@ -0,0 +1,167 @@
+						other: translate( 'Other…' ),
+					} } />
+				{ 'other' === contentsType && <TextField
+					id={ packageId + '_contentsExplanation' }
+					title={ translate( 'Details' ) }
+					value={ contentsExplanation || '' }
+					updateValue={ props.setContentsExplanation }
+					error={ errors.contentsExplanation } /> }
+			</div>
+
+			<div className="customs-step__restriction-type">
+				<Dropdown
+					id={ packageId + '_restrictionType' }
+					title={ translate( 'Restriction type' ) }
+					value={ restrictionType || 'none' }
+					updateValue={ props.setRestrictionType }
+					valuesMap={ {
+						none: translate( 'None' ),
+						quarantine: translate( 'Quarantine' ),
+						sanitary_phytosanitary_inspection: translate( 'Sanitary / Phytosanitary inspection' ),
+						other: translate( 'Other…' ),
+					} } />
+				{ 'other' === restrictionType && <TextField
+					id={ packageId + '_restrictionExplanation' }
+					title={ translate( 'Details' ) }
+					value={ restrictionExplanation || '' }
+					updateValue={ props.setRestrictionExplanation }
+					error={ errors.restrictionExplanation } /> }
+			</div>
+		</div>
+
+		<TextField
+			id={ packageId + '_itn' }
+			title={
+				<span>{ translate( 'ITN' ) } (<a href="https://pe.usps.com/text/imm/immc5_010.htm"
+					target="_blank" rel="noopener noreferrer">{ translate( 'more info' ) }</a>)</span>
+			}
+			value={ itn || '' }
+			updateValue={ props.setITN }
+			error={ errors.itn } />
+
+		<div className="customs-step__item-rows">
+			<ItemRowHeader />
+			{ uniq( map( items, 'product_id' ) ).map( productId => (
+				<ItemRow
+					key={ productId }
+					productId={ productId }
+					packageId={ packageId }
+					siteId={ siteId }
+					orderId={ orderId } />
+			) ) }
+		</div>
+	</div>;
+};
+
+PackageRow.propTypes = {
+	siteId: PropTypes.number.isRequired,
+	orderId: PropTypes.number.isRequired,
+	packageId: PropTypes.string.isRequired,
+	errors: PropTypes.object,
+	contentsType: PropTypes.oneOf( [ 'merchandise', 'documents', 'gift', 'sample', 'other' ] ),
+	contentsExplanation: PropTypes.string,
+	restrictionType: PropTypes.oneOf( [ 'none', 'quarantine', 'sanitary_phytosanitary_inspection', 'other' ] ),
+	restrictionExplanation: PropTypes.string,
+	abandonOnNonDelivery: PropTypes.bool,
+	itn: PropTypes.string,
+	items: PropTypes.arrayOf(
+		PropTypes.shape( {
+			product_id: PropTypes.number.isRequired,
+		} )
+	).isRequired,
+	setContentsType: PropTypes.func.isRequired,
+	setContentsExplanation: PropTypes.func.isRequired,
+	setRestrictionType: PropTypes.func.isRequired,
+	setRestrictionExplanation: PropTypes.func.isRequired,
+	setAbandonOnNonDelivery: PropTypes.func.isRequired,
+	setITN: PropTypes.func.isRequired,
+};
+
+const mapStateToProps = ( state, { orderId, siteId, packageId } ) => {
+	const loaded = isLoaded( state, orderId, siteId );
+	const shippingLabel = getShippingLabel( state, orderId, siteId );
+
+	return {
+		...shippingLabel.form.packages.selected[ packageId ],
```

Comment:
Let's be explicit about which props we're using here, so that it's easy to see at a glance.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/3d01c68006308df411def1cda6b70f59aba89d7d